### PR TITLE
Use asset pipeline path helpers for virtual tour files

### PR DESCRIPTION
**Pull request by h-lame**
The SWF and XML files are in the asset pipeline already, but we refer to
them directly.  This means the tour only works on dev (where assets don't
get compiled into hash urls) or on production (where the unadorned files
have been uploaded to the whitehall frontend machines directly).

If we use asset_path helpers instead the tour should work in all
environments.

Spotted while trying to test https://trello.com/c/ni5o7K2J/70-changes-to-10-downing-street-history-page on integration

### CODE REVIEW

**Comment by h-lame** [comment]
Marked as WIP as the only way to test if this works is to deploy and see what breaks.  Will remove WIP once we have a version that actually works.

**Comment by h-lame** [comment]
Ofc as soon as I add the WIP tag the version that's up proves to be the one that works.  Ready to review now.

File: app/assets/javascripts/tour/garden.xml.erb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<krpano version="1.0.8.14">
+
+	<view hlookat="0" vlookat="0" maxpixelzoom="1.0" limitview="auto" />
+	<include url="<%= asset_path("tour/skin/defaultskin.xml") %>" />
+
+  <view fisheye="0" limitview="range" hlookatmin="-180" hlookatmax="180" vlookatmin="-90" vlookatmax="90" maxpixelzoom="1.0" fovmax="90" fov="90" hlookat="78" vlookat="10"/>
+  <test url="<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/garden.tiles/preview.jpg" />
```
Comment (gpeng) [inline app/assets/javascripts/tour/garden.xml.erb]: Now this is `.erb` would it be worth extracting a bit more of this path into a helper to reduce the `<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/` repetition?

Comment (h-lame) [inline app/assets/javascripts/tour/garden.xml.erb]: Oh, yeah.  Maybe there already is a helper for this - I'll check.

Comment (h-lame) [inline app/assets/javascripts/tour/garden.xml.erb]: There's no existing helper I can use, but I can use a route helper and replace:

```erb
<test url="<%= Whitehall.public_asset_host %>/government/uploads/uploaded/number10/garden.tiles/preview.jpg" />
```

with:

```erb
<% url_helpers = Rails.application.routes.url_helpers %>
<test url="<%= url_helpers.public_upload_path("uploaded/number10/garden.tiles/preview", extension: "jpg") %>" />
```

I'm 50:50 on that being better though, thoughts?

Comment (gpeng) [inline app/assets/javascripts/tour/garden.xml.erb]: Sorry @h-lame... Didn't realise this was waiting on me. Yeah... I don't see any particular benefit in either approach over the other. Let's leave the one you've done.

**Pull request merged**